src/box/applier.cc

```diff
@@ -861,9 +861,8 @@ applier_fetch_snapshot(struct applier *applier)
 	struct iostream *io = &applier->io;
 	struct xrow_header row;
 
-	struct fetch_snapshot_request req = {
-		.version_id = tarantool_version_id(),
-	};
+	struct fetch_snapshot_request req;
+	req.version_id = tarantool_version_id();
```

Comment:
Why change it?

Reply:
In order not to initialize all rows explicitly. `checkpoint_vclock` and `checkpoint_lsn` remains unitialized

Reply:
Do we really need them to remain uninitialized? What's wrong with setting them to 0?

Reply:
Now they're intiialized